modules/administration-guide/pages/finding-the-fully-qualified-domain-name-fqdn.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: PROCEDURE
+:description: Finding the fully qualified domain name (FQDN)
+:keywords: administration-guide, FQDN, URL
+:navtitle: Finding the fully qualified domain name (FQDN)
+//:page-aliases:
+
+[id="finding-the-fully-qualified-domain-name-fqdn"]
+= Finding the fully qualified domain name (FQDN)
+
+You can get the fully qualified domain name (FQDN) of your organization's instance of {prod-short} on the command line or in the OpenShift web console.
+
+TIP: You can find the FQDN for your organization's {prod-short} instance in the *Administrator* view of the OpenShift web console as follows. Go to *Operators* -> *OperatorHub* -> *Installed Operators* -> *{prod} instance Specification* -> *{prod-checluster}* -> *{prod} URL*.
+
+
+.Prerequisites
+
+* An active `oc` session with administrative permissions to the OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
+* `{prod-cli}`. See xref:installing-the-chectl-management-tool.adoc[].
```

Comment:
Suggestion for better clarity:
* You have an active `oc` session with administrative permissions to the OpenShift Container Platform cluster. For more details, see....
*  You have installed the `{prod-cli}` management tool. For more details, see....

We can use an attribute for OpenShift Container Platform like {product-title}.

Reply:
Thank you for this suggestion.